app/views/publishers/organisations/_vacancy_draft.html.slim

```diff
@@ -13,4 +13,4 @@
       ul.actions.govuk-list
         li = govuk_link_to(t("jobs.edit_link"), vacancy.edit_path)
         li = govuk_link_to(t("jobs.copy_link"), vacancy.copy_path, method: :get)
-        li = govuk_link_to(t("jobs.delete_link"), vacancy.delete_path, method: :delete, data: { confirm: t("jobs.are_you_sure", job_title: vacancy.job_title) })
+        li = button_to(t("jobs.delete_link"), vacancy.delete_path, method: :delete, data: { confirm: t("jobs.are_you_sure", job_title: vacancy.job_title) }, class: "govuk-delete-link")
```

Comment:
Why don't we use:

https://github.com/DFE-Digital/govuk-components/blob/master/app/helpers/govuk_link_helper.rb#L10

Reply:
It makes the button look like a `govuk-button`, but we don't want that

Reply:
ah ok. fair enough